apps/src/templates/instructions/InstructionsCSF.jsx

```diff
@@ -552,169 +331,37 @@ class InstructionsCSF extends React.Component {
             ],
```

Comment:
Suggestion: move the style defined on L327-332 out of the tag (up into the render function?)